Gemfile

```diff
@@ -6,7 +6,13 @@ source "https://rubygems.org"
 gemspec
```

Comment:
Just a question: why do we have some dependencies in `.gemspec` file and some of them in `Gemfile`?

Reply:
I ended up having application dependencies in `.gemspec` and support dependencies in groups in `Gemfile`. `bundle gem` added automatically `rake` and `rspec` in the Gemfile, so, not sure if it is a valid approach